Client/src/app/albums-container/albums-container.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Album } from '../models/album';
+
+@Component({
+  selector: 'app-albums-container',
+  templateUrl: './albums-container.component.html',
+  styleUrls: ['./albums-container.component.css'],
```

Comment:
Let's remove this line and corresponding css file if it's empty. Here and in every component